hugegraph-core/src/main/java/com/baidu/hugegraph/type/Indexifiable.java

```diff
@@ -23,7 +23,7 @@
 
 import com.baidu.hugegraph.backend.id.Id;
 
-public interface Indexfiable {
+public interface Indexifiable {
```

Comment:
API break?

Reply:
@javeme maybe we should check the influence in other repository, however we need correct it before move to apache

and thanks @jsoref for reminding, is it also an automatic rule in your tool?

Reply:
The tool recognized this as a non-word. The correction here was manual (Google Sheets guessed `Index Fiable`).

Picking correct words is generally left as the responsibility of a human, although for large lists, I'm able to leverage Google Sheets corrections for some portion of items.

Reply:
I think "Indexable" is a more appropriate word here

Reply:
Oh, I should note that the places where I comment about api breaks are purely manual, it's my way of saying "I'm a human, not a robot, I do understand that changes have consequences". They also aren't necessarily exhaustive (I tend to do this changes while exhausted...).